packages/pn-personafisica-login/src/pages/success/Success.tsx

```diff
@@ -2,11 +2,14 @@ import { useCallback, useEffect, useMemo } from 'react';
 
 import { AppRouteParams, sanitizeString } from '@pagopa-pn/pn-commons';
 
+import { PFLoginEventsType } from '../../models/PFLoginEventsType';
 import { getConfiguration } from '../../services/configuration.service';
+import PFLoginEventStrategyFactory from '../../utility/MixpanelUtils/PFLoginEventStrategyFactory';
 import { storageAarOps } from '../../utility/storage';
 
 const SuccessPage = () => {
   const { PF_URL } = getConfiguration();
+  const IDP = sessionStorage.getItem('IDP');
```

Comment:
Move this, the event trigger and the removeItem into the useEffect, after the calcRedirectUrl();